code/model/editableformfields/EditableFormField.php

```diff
@@ -984,4 +1003,62 @@ public function EffectiveDisplayRules()
         }
         return $this->DisplayRules();
     }
+
+	public function FormatDisplayRules()
+	{
+		$field = $this;
+		$holderSelector = $field->getSelectorOnly();
+		$result = array(
+			'targetFieldID' => $holderSelector,
+			'conjunction'   => $field->DisplayRulesConjunctionNice(),
+			'selectors'     => array(),
+			'events'        => array(),
+			'operations'    => array(),
+			'initialState'  => $this->ShowOnLoadNice(),
+			'view'  => array(),
+			'opposite'  => array(),
+		);
+
+		// Check for field dependencies / default
+		/** @var EditableCustomRule $rule */
+		foreach($field->EffectiveDisplayRules() as $rule)
+		{
+			// Get the field which is effected
+			/** @var EditableFormField $formFieldWatch */
+			$formFieldWatch = EditableFormField::get()
+											   ->byId($rule->ConditionFieldID);
```

Comment:
Okay. Found the PHPStorm setting for this. In Preferences > Code Style > PHP > Wrapping and Braces > Chained Method Calls - change from _Wrap Always_ to _Wrap if long_.